_shows/20_21/the_show_must_go_on.md

```diff
@@ -0,0 +1,79 @@
+---
+# Check out the Show Docs for more information
+# -- https://history.newtheatre.org.uk/docs/show/
+# Remove or comment any lines that are not being used
+
+title: The Show Must Go On(Line)
+playwright: Charlie Bellwood, Oliver Binns and Nicholas Landon
+student_written: true
+period: Autumn
+season: Online
```

Comment:
Online unscripted show… @BenWoodford76 / @GingertronMk1 - which season do we reckon?

Reply:
Unscripted I Reckon